projects/storefrontlib/src/lib/product/components/product-details/product-summary/product-summary.component.ts

```diff
@@ -44,7 +39,52 @@ export class ProductSummaryComponent {
     );
   }
 
-  launchReview() {
-    this.openReview.emit();
+  // Get Tabs Component if exists on page
+  private getTabsComponent(): Element {
+    const elements: HTMLCollectionOf<Element> = document.getElementsByClassName(
+      'Tabs'
+    );
+
+    return elements.length > 0 ? elements[0] : null;
   }
+
+  // Get Reviews Tab if exists on page
+  private getReviewsTab(): HTMLElement {
```

Comment:
It will be nice to have property type Observable<boolean>, allows us to hide "Show reviews" button in case, when Reviews are not displayed in tabs